schema.sql

```diff
@@ -0,0 +1,21 @@
+CREATE DATABASE my_catalog;
+
+CREATE TABLE author(
+  ID SERIAL PRIMARY KEY,
+  first_name VARCHAR(30),
+  last_name VARCHAR(30),
+);
+
+CREATE TABLE book(
```

Comment:
The table is called game, kindly correct that mistake 😅

Reply:
:sweat_smile: What a mistake